webapi/Models/Response/BotResponsePrompt.cs

```diff
@@ -37,7 +60,7 @@ public class BotResponsePrompt
     /// Relevant additional knowledge extracted using a planner.
     /// </summary>
     [JsonPropertyName("externalInformation")]
-    public string ExternalInformation { get; set; } = string.Empty;
+    public SemanticDependency<StepwiseThoughtProcess> ExternalInformation { get; set; }
```

Comment:
Is BotResponsePrompt currently used for other context types? Will this break them in the time between checking this in and adding the others?

Reply:
No, it's only used for the prompt view in the webapp and I've made that backwards compatible with string types too (since we were just returning the raw rendered string to show in the prompt dialog before)